client/app/dedicated/server/dedicated-server.service.js

```diff
@@ -1328,6 +1328,101 @@ angular
       }).then(details => details, () => null);
     };
 
+    this.getBareMetalPrivateBandwidthOptions = function (productId, existingBandwidth) {
+      return OvhHttp.get('/order/upgrade/baremetalPrivateBandwidth/{serviceName}', {
```

Comment:
I think tou should add services in [`ovh-api-services`](https://github.com/ovh-ux/ovh-api-services/blob/master/src/order/upgrade/order-upgrade.service.js) instead of hoard-coding `OvhHttp` call.